neqo-qpack/src/fuzz.rs

```diff
@@ -0,0 +1,88 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Fuzzing support for QPACK.
+
+#[cfg(feature = "build-fuzzing-corpus")]
```

Comment:
The conditional compilation here hurts a little.  There are two things here, could they be added to modules (and re-exported) to clarify it a bit?

```rust
#[one]
pub(crate) use one::{...};
#[two]
pub(crate) use two::{...};

#[one]
mod one {

}

#[two]
mod two {
}
```